File: src/main/js/werkboek/week1/les1/1_3.js
class Student {
    constructor(nm){
        this.name = nm;
    }  // Javascript doet in principe niet moeilijk over public/private
}



let student1 = new Student("Jan")
let student2 = new Student("Wim")

console.log(student1)
console.log(student1.name)
console.log(student2)
console.log(student2.name)

// In Javascript ziet het er standaard al best aardig uit!

File: src/main/python/werkboek/week1/les1/1_3.py
class Student:
    def __init__(self, name):
        self.name = name  # Python doet in principe niet moeilijk over public/private


student1 = Student("Jan")
student2 = Student("Wim")

print(student1)
print(student1.name)
print(student2)
print(student2.name)

# Valt ook in Python een beetje tegen nietwaar?
